### PR TITLE
Improve sticky note text rendering

### PR DESCRIPTION
**Pull request by griffi-gh**
Shoves (up to) 64 notes into a single 1024x1024/2048x2048 texture!    
  
Everything is configurable! (Note width, height, font scale, texture width/height, slot count, non-square textures *should* be supported)  

should be good enough (tm) to merge, but please test thoroughly.
  
This also adds a new "Sticky note resolution" option (which is enabled *by default* on PC, and disabled on Android)
  
This is not a complete rewrite, and it still does some weird things under the hood...

Btw, I disabled layer switching I added in the #107, mostly just for testing, but it didn't work wery well anyway. :p   
basically just a visual change.

### CODE REVIEW

**Comment by griffi-gh** [comment]
Got to "usable" stage:

My sticky implementation:
principia_xxIwYsjCVL


Latest beta:

principia_RtxYwxE5ol

There are still some major issues though

**Comment by griffi-gh** [comment]
principia_wN2QVrfkV3

decided to 2x the resolution on pc (and keep the same resolution on mobile...)

**Comment by griffi-gh** [comment]
principia_ESF9Lh2nnL  
  
this is actually a huge improvement.  
0 font-sized notes are not blurry anymore

**Comment by griffi-gh** [comment]
I'll add a setting for it

**Comment by rollerozxa** [comment]
Is there any reason the sticky note resolution can't just be the higher resolution without a setting?

**Comment by griffi-gh** [comment]
> Is there any reason the sticky note resolution can't just be the higher resolution without a setting?

because it significantly slows down level loading if there are a lot of notes on low-end devices?

**Comment by griffi-gh** [comment]
probably should be merged into a single "Texture quality" setting

**Comment by griffi-gh** [comment]
wait we already have that right?
